examples/simulation/Tutorial2/macros/compare_seed_value.C

```diff
@@ -0,0 +1,20 @@
+int compare_seed_value(TString filename, UInt_t initial_seed) {
```

Comment:
Could you add a copyright header and basic set of includes, please?

```suggestion
/********************************************************************************
 *    Copyright (C) 2023 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH    *
 *                                                                              *
 *              This software is distributed under the terms of the             *
 *              GNU Lesser General Public Licence (LGPL) version 3,             *
 *                  copied verbatim in the file "LICENSE"                       *
 ********************************************************************************/

#include <TFile.h>
#include <iostream>
#include <memory>

int compare_seed_value(TString filename, UInt_t initial_seed) {
```